tests/integration/docker_utils/test_docker.py

```diff
@@ -729,6 +729,52 @@ def test_get_logs_non_existent_container(self, docker_client: ContainerClient):
             "container_hopefully_does_not_exist", safe=True
         )
 
+    def test_get_logs(self, docker_client: ContainerClient):
+        container_name = _random_container_name()
+        try:
+            docker_client.run_container(
+                "alpine",
+                name=container_name,
+                detach=True,
+                command=["env"],
+            )
+
+            logs = docker_client.get_container_logs(container_name)
+            assert "PATH=" in logs
+            assert "HOSTNAME=" in logs
+            assert "HOME=/root" in logs
+
+        finally:
+            docker_client.remove_container(container_name)
+
+    def test_stream_logs_non_existent_container(self, docker_client: ContainerClient):
+        with pytest.raises(NoSuchContainer):
+            docker_client.get_container_logs("container_hopefully_does_not_exist", safe=False)
```

Comment:
Test is called test_stream_logs_non_existent_container, but executes get_container_logs instead of stream_container_logs?

Reply:
oops! copy pasted and never looked back :sweat_smile: